SmartDeviceLink/SDLCancelInteraction.h

```diff
@@ -71,28 +71,28 @@ NS_ASSUME_NONNULL_BEGIN
 
  @return A SDLCancelInteraction object
  */
-- (instancetype)initWithAlert;
++ (instancetype)alert;
 
 /**
  Convenience init for dismissing the currently presented slider.
 
  @return A SDLCancelInteraction object
  */
-- (instancetype)initWithSlider;
++ (instancetype)slider;
 
 /**
  Convenience init for dismissing the currently presented scrollable message.
 
  @return A SDLCancelInteraction object
  */
-- (instancetype)initWithScrollableMessage;
++ (instancetype)scrollableMessage;
 
 /**
  Convenience init for dismissing the currently presented perform interaction.
 
  @return A SDLCancelInteraction object
  */
-- (instancetype)initWithPerformInteraction;
++ (instancetype)performInteraction;
```

Comment:
This needs an `NS_SWIFT_NAME` because for some reason swift is interpreting it as `.perform()` and it should be `.performInteraction()`

Reply:
added